kvclient_ops.go

```diff
@@ -52,11 +53,14 @@ func kvClient_SimpleCall[Encoder any, ReqT any, RespT any](
 	req ReqT,
 ) (RespT, error) {
 	resulter := allocSyncCrudResulter()
+	var calledBack uint32
 
 	pendingOp, err := execFn(o, c.cli, req, func(resp RespT, err error) {
-		resulter.Ch <- syncCrudResult{
-			Result: resp,
-			Err:    err,
+		if atomic.CompareAndSwapUint32(&calledBack, 0, 1) {
```

Comment:
We should probably have an `} else { panic() }` since this is a serious logic error on our part if it occurs.  We may also want to have this check only occur when we are running with --race or something, since assuming gocbcorex is written correctly, this should never occur.

Reply:
I think that logger.DPanic() would probably the answer there.